### PR TITLE
use exported component

### PR DESCRIPTION
**Pull request by d4hines**
refactor: #id [22728]

[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/22728/details)

**Description of change**
Adds a SingleInputDialog component, copied from the seed project.

**Description of testing**
I verified each of the requirements in the below doc, using the New Workspace dialog as the test case.

https://docs.google.com/document/d/1SV-JIC7D1WxtLHIVxieaB4SB7c1nod5JXMe6MkGYt7o/edit

### CODE REVIEW

File: src-built-in/components/singleInputDialog/src/app.jsx [hunk 1]
```diff
@@ -6,140 +6,15 @@ import React from "react";
...
-			</div>
-		</FinsembleDialog>);
-	}
-}
+import { SingleInputDialog } from "@chartiq/finsemble-ui/lib/components/singleInputDialog/singleInputDialog";
 
 //render component when FSBL is ready.
-if (window.FSBL && FSBL.addEventListener) { FSBL.addEventListener("onReady", FSBLReady); } else { window.addEventListener("FSBLReady", FSBLReady) }
+
 function FSBLReady() {
```
Comment (zona-ciq) [inline src-built-in/components/singleInputDialog/src/app.jsx]: We can hide this FSBLReady function inside finsemble UI and only renders it when FSBL is ready

Comment (d4hines) [reply src-built-in/components/singleInputDialog/src/app.jsx]: Done.

File: src-built-in/components/singleInputDialog/src/app.jsx [hunk 2]
```diff
@@ -6,140 +6,8 @@ import React from "react";
...
-		this.bindCorrectContext();
-		this.state = {
-			inputLabel: "No question.",
-			affirmativeResponseLabel: "Yes",
-			/** Our input uses the HTML autoFocus attribute. This only fires when the element is rendered for the 1st time. Since we re-use the dialogs, the 2nd time the dialog is shown, autoFocus will not trigger. To force it to trigger, we change this boolean when a response is sent back to the opener, which unmounts the element from the dom. The next time the dialog is used, we will render the element again, and autoFocus will trigger. */
-			renderInput: false
-		};
-		document.body.addEventListener("keydown", this.handleKeydownOnBody);
-	}
+import { SingleInputDialog } from "@chartiq/finsemble-ui/lib/components/singleInputDialog/singleInputDialog";
```
Comment (zona-ciq) [inline src-built-in/components/singleInputDialog/src/app.jsx]: This can just be `import { SingleInputDialog } from "@chartiq/finsemble-ui/lib/components`

Comment (d4hines) [reply src-built-in/components/singleInputDialog/src/app.jsx]: Done!

**Pull request merged**